setup.py

```diff
@@ -17,58 +17,35 @@ def find_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 
-def read(*parts):
-    return codecs.open(os.path.join(os.path.dirname(__file__), *parts),
-                       encoding='utf8').read()
-
-
-try:
-    bytes
-except NameError:
-    bytes = str
-
+def get_version(package):
+    """
+    Return package version as listed in `__version__` in `init.py`.
+    """
+    matches = re.search("__version__[ ]+=[ ]+['\"](?P<version>[^'\"]+)['\"]",
```

Comment:
Hum... Maybe you should use a raw string here : `r"__version__[\s]+=[\s]+['\"](?P<version>[^'\"]+)['\"]"`

Reply:
Okay, replaced